### PR TITLE
Update convolutional.py

### PR DESCRIPTION
**Pull request by ymodak**
fixes https://github.com/tensorflow/tensorflow/issues/48137

### CODE REVIEW

File: keras/layers/convolutional.py [hunk 1]
```diff
@@ -579,8 +579,8 @@ class Conv2D(Conv):
       value != 1 is incompatible with specifying any `dilation_rate` value != 1.
     padding: one of `"valid"` or `"same"` (case-insensitive).
       `"valid"` means no padding. `"same"` results in padding with zeros evenly
-      to the left/right or up/down of the input such that output has the same
-      height/width dimension as the input.
+      to the left/right or up/down of the input. The output has the same
+      height/width dimension as the input when `"same"` and `strides=(1,1)`.
```
Comment (fchollet) [inline keras/layers/convolutional.py:583]: "When `padding="same"` and `strides=1`, the output has the same size as the input."

File: keras/layers/convolutional.py [hunk 2]
```diff
@@ -579,8 +579,8 @@ class Conv2D(Conv):
       value != 1 is incompatible with specifying any `dilation_rate` value != 1.
     padding: one of `"valid"` or `"same"` (case-insensitive).
       `"valid"` means no padding. `"same"` results in padding with zeros evenly
-      to the left/right or up/down of the input such that output has the same
-      height/width dimension as the input.
+      to the left/right or up/down of the input. When padding="same" and
```
Comment (fchollet) [inline keras/layers/convolutional.py:582]: Use backticks around codde keywords

**Review by fchollet — APPROVED** [review]

**Review by fchollet — APPROVED** [review]
